### ISSUE TITLE
Could you please provide hubert iteration 1 model?

### ISSUE THREAD

**Issue by mechanicalsea** [facebookresearch/fairseq#4042]
The hubert base, large, or xlarge models are released in huggingface. But the iteration 1 of hubert pre-training that provide label for iteration 2 pre-training can not be found anywhere.

If okay, could you release the iteration 1 of hubert model, which is helpful to facilitate hubert re-implementation and iteration 2's labels generation.

Wish the suggestion can be considered. Thank you.

**Comment by wnhsu** [comment]
Hi @mechanicalsea , unfortunately the 1st iteration BASE checkpoint was removed by accident... 
However, the large and extra large model was trained using the 2nd iteration BASE model label, which is the one provided in the README.md. We have also upload the k-means model used for quantization. hope this helps.

**Comment by mechanicalsea** [comment]
@wnhsu Thanks for your response. A part of a label, the test scripts of labels, and the k-means model are provided as mentioned in the README.md. It is helpful to verify our reproduced 1st iteration BASE model and help obtain the labels of HuBERT. 

Is the `L9 km500` of BASE-it2-layer9?

As provided as README.md, obtaining the HuBERT BASE model's label still requires re-implementation of 1st iteration HuBERT BASE. It may make verifying the 1st iteration HuBERT BASE challenging, because there exists randomness in the MFCC k-means model and the BASE model pre-training processing of the 1st iteration. 

If permitted, could you provide the 2nd iteration BASE model label, or the LibriSpeech dev/test performance of the 1st iteration BASE model (only the results of dev-other is provided in Table V of the paper with 100k steps pre-training and n-gram language model), or something helpful to verify the 1st iteration HuBERT BASE model and the HuBERT BASE labels at the 2nd iteration?

We tested the reproduced model as the performance of the ASR dataset. As the pipeline provided in the README.md, we obtained the 1st iteration HuBERT BASE WER results on the 10 hours ASR dataset and on the 100 hours ASR dataset as follows,

|Dataset|dev-clean|dev-other|other-clean|test-other|
|:---|---:|---:|---:|---:|
|10 hours ASR|14.071%|23.204%|14.343%|23.612%|
|100 hours ASR|8.086%|18.209%|8.42%|17.716%|

where the dataset was used as the HuBERT paper and **no language model** was applied. But, the pre-training process is not as expected in the metric of `correct_u`. We re-generated the MFCC label and concluded a similar result.

> Hi @mechanicalsea , unfortunately the 1st iteration BASE checkpoint was removed by accident... However, the large and extra large model was trained using the 2nd iteration BASE model label, which is the one provided in the README.md. We have also upload the k-means model used for quantization. hope this helps.

**Comment by orsdanilo** [comment]
Kind of late here, but I'm interested in this as well.
Would it be possible to obtain the pseudo labels used to train the 2nd iteration of HuBERT?
